lib/osm_nominatim.dart

```diff
@@ -65,6 +65,7 @@ class Nominatim {
     List<String>? excludePlaceIds,
     int limit = 10,
     ViewBox? viewBox,
+    String? nominatimServer
```

Comment:
I'd name this `host` and set the default value in the parameters like this: `String host = 'nominatim.openstreetmap.org',`